safe-global/safe-wallet-web
# feat: add promo widget for swaps [SWAP-87]

## What it solves
Displays a swaps promo block(that can be hidden) on the dashboard. 
Right now we are loading a .png, but @TanyaEfremova would give us an svg.

Resolves #

## How this PR fixes it

- Adds new Swap promo widget on the dashboard
- Adjusts the `Chip` style to match Figma design

Other changes:
- Adjusts the Queue counter style to match Figma design
- Adjusts the sidebar badges to match Figma design

## How to test it

1. Open the dashboard and observe the new widget
2. Press "Try it now" and be navigated to the swaps page
3. Press "Don't show again", the widget should permanently disappear

## Screenshots

<img width="1278" alt="Screenshot 2024-05-31 at 10 52 43" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/e4ec8022-dfcb-474a-ba7d-3f1d40f2145a">
<img width="247" alt="Screenshot 2024-05-31 at 10 52 58" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/30972ec8-8d75-4328-bccc-42f595c64701">
<img width="411" alt="Screenshot 2024-05-31 at 10 53 15" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/28e5f4b4-e573-461c-95f4-54c7a55ed4be">


## Checklist
